docs/development.md

```diff
@@ -46,20 +46,31 @@ kubectl create -f deploy/crds/gcp_v1alpha1_projectreference_crd.yaml
 
 Make sure you have the [operator-sdk](https://github.com/operator-framework/operator-sdk/releases) binary in your `$PATH` and run it locally:
 
-```zsh
+```shell
 $ operator-sdk run --local --namespace gcp-project-operator
 ```
 
 You will see some initialization logs.
 The Operator will remain _idle_ after that, waiting for `ProjectClaim` resources to be present in the cluster.
 
+You can change the verbosity of the logs, by passing the `--zap-level` flag as part of the `--operator-flags`:
+
+```shell
+run --local --namespace gcp-project-operator --operator-flags --zap-level=99
+```
+
+* Level 1: ProjectReference
+* Level 2: ProjectClaim
+* Level 3: gcpclient
+* Level 4: operator-sdk framrwork logs (golang version, etc)
```

Comment:
can we have these as constants so we don't have to look it up each time we use it?

Reply:
Added the constants